src/pages/dataTableUtilities/getIndicatorTableData.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```

Comment:
whats that about

Reply:
7abbb235.

![cookie](https://user-images.githubusercontent.com/43223637/72314980-e7ad4080-36f5-11ea-89d8-5e8f091cfb6a.gif)